docs/UserGuide.adoc

```diff
@@ -234,12 +242,86 @@ Format: `warn INDEX q/MIN_QUANTITY`
 
 Example:
 
-* `warn 1 q/100` +
+* `warn 5 q/100` +
 This sets the minimum quantity threshold for good at index 1 with an quantity of 100.
 
+.Demo for `warn` command
+image::UG/warn.png[width="790"]
+
 [NOTE]
 When a new good is added into the inventory, its minimum quantity threshold is set at 0.
 
+=== Locating transactions by search criteria:
+
+Display list of transactions that fulfills the given search criteria.
+
+3 types of search criteria:
+
+. transaction type
+. `Name` of `supplier`
+. `GoodName` of `Good`
```

Comment:
Should user know the class names? It seems like breaking abstraction

Reply:
`=== Adding a supplier: `add`

Adds a supplier to the inventory manager +
Format: `add n/NAME c/CONTACT_NUMBER e/EMAIL a/ADDRESS [gp/GOOD PRICE]...``

then does this breaks the abstraction as well?